pkg/scheduler/objects/template/template.go

```diff
@@ -44,7 +45,7 @@ func FromConf(template *configs.ChildTemplate) (*Template, error) {
 		return true
 	}
 
-	if template == nil || (isMapEmpty(template.Properties) && isMapEmpty(template.Resources.Guaranteed) && isMapEmpty(template.Resources.Max)) {
+	if template == nil || (template.MaxApplications == 0 && isMapEmpty(template.Properties) && isMapEmpty(template.Resources.Guaranteed) && isMapEmpty(template.Resources.Max)) {
```

Comment:
Could you refactor this part and `isMapEmpty` to a separate method for readability?
```
if template == nil || template.IsEmpty() {
  return nil, nil
}

...

func (t *Template) IsEmpty() bool {
...
}

func isMapEmpty(m map[string]string) bool {
...
}
```

Reply:
Already address comments in the latest comment, many thanks !